### PR TITLE
[Trivial] Slightly adjust some README.md files

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 Dash Core staging tree 0.14.1
-===============================
+=============================
```
Comment (nmarley) [inline README.md]: Why don't we just add a `#` at the beginning of this header and remove the `===` entirely? It achieves the same effect in Markdown and doesn't require re-aligning the `===` underneath every time.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 `master:` [![Build Status](https://travis-ci.org/dashpay/dash.svg?branch=master)](https://travis-ci.org/dashpay/dash) `develop:` [![Build Status](https://travis-ci.org/dashpay/dash.svg?branch=develop)](https://travis-ci.org/dashpay/dash/branches)
 
 https://www.dash.org
 
 
 What is Dash?
-----------------
+-------------
```
Comment (nmarley) [inline README.md]: Same, can use `##` for the H2 header.

File: README.md (hunk 1 above)
Comment (PastaPastaPasta) [reply README.md]: sounds like another pr to me :)

Comment (UdjinM6) [reply README.md]: `#` and `====` are used interchangeably in various other `*.md` files all over the repo. Should swap them all at once if we decide to do so imo, but I would rather just leave it as is tbh.

Comment (nmarley) [reply README.md]: Agree this is a nit, and swapping them all at once makes sense.

**Pull request merged**